appcelerator/titanium_mobile
# [TIMOB-25667] Android: Added TableView "scrollable" property support

**JIRA:** https://jira.appcelerator.org/browse/TIMOB-25667

**Test:**
1. Build and run the below code on an Android device.
2. Verify that you can scroll the TableView up/down.
3. Tap the "Disable Scrolling" button in the bottom-right corner.
4. Verify that you cannot scroll the TableView.
5. Tap the "Enable Scrolling" button in the bottom-right corner.
6. Verify that you can scroll the TableView up/down.

```javascript
var window = Ti.UI.createWindow();
var tableData = [];
for (var index = 1; index <= 100; index++) {
	var row = Ti.UI.createTableViewRow({ title: "Row " + index.toString() });
	tableData.push(row);
}
var tableView = Ti.UI.createTableView(
{
	data: tableData,
	scrollable: true,
});
window.add(tableView);
function createButtonText() {
	return tableView.scrollable ? "Disable Scrolling" : "Enable Scrolling";
}
var button = Ti.UI.createButton(
{
	title: createButtonText(),
	right: "5dp",
	bottom: "5dp",
});
button.addEventListener("click", function(e) {
	tableView.scrollable = !tableView.scrollable;
//	tableView.setScrollable(!tableView.getScrollable());
	button.title = createButtonText();
});
window.add(button);
window.open();
```
